build.sbt

```diff
@@ -139,12 +139,23 @@ lazy val node = (project in file("node"))
     maintainer in Linux := "Pyrofex, Inc. <info@pyrofex.net>",
     packageSummary in Linux := "RChain Node",
     packageDescription in Linux := "RChain Node - the RChain blockchain node server software.",
+     linuxPackageMappings += {
+      val file = baseDirectory.value / "rnode.service"
+      packageMapping( (file -> "/lib/systemd/system/rnode.service") )
+    },
     /* Debian */
+    packageSummary in Debian := "RChain Node",
+    packageDescription in Debian := "RChain Node - the RChain blockchain node server software.",
     debianPackageDependencies in Debian ++= Seq("openjdk-8-jre-headless", "bash (>= 2.05a-11)", "libsodium18 (>= 1.0.8-5)"),
-    /* Redhat */
+    linuxPackageMappings in Debian := linuxPackageMappings.value,
+   /* Redhat */
+    packageSummary in Rpm := "RChain Node",
+    packageDescription in Rpm := "RChain Node - the RChain blockchain node server software.",
     rpmVendor := "rchain.coop",
     rpmUrl := Some("https://rchain.coop"),
     rpmLicense := Some("Apache 2.0"),
+    packageArchitecture in Rpm := "noarch",
```

Comment:
Below this you should add:

```scala
maintainerScripts in Rpm := maintainerScriptsAppendFromFile((maintainerScripts in Rpm).value)(
  RpmConstants.Post -> (sourceDirectory.value / "rpm" / "scriptlets" / "post")
)
```

This is because `sbt-native-packager` is broken and doesn't seem to add this as a a default (even though the documentation says it should).